src/02-data-types.js

```diff
@@ -95,7 +107,13 @@ function checkIfCharacterIsInString(text, character) {}
  *  isEven(11);
  *  //> false
  */
-function isEven(num) {}
+function isEven(num) {
+  if (num%2 === 0){
```

Comment:
we can just return `num % 2 === 0`. also let's add some spaces between our operator and values.